### PR TITLE
completed feature 1 can view random recommendation - YoonHwan Cho

### PR DESCRIPTION
**Pull request by YoonHCho**
completed feature 1 can view random recommendation - YoonHwan Cho
AJAX Feature 1 of 2
AJAX Feature 2 of 2

### CODE REVIEW

**Comment by YoonHCho** [comment]
address what was mentioned on slack
1. removed favorite and genre selection
2. changed on figma as well.
3. added filter so that the adult content will not load (tried refresh 10 times)

**Comment by YoonHCho** [comment]
fixed repeated random anime.
font is roboto in figma and in project
AJAX Feature 1

File: index.html [hunk 1]
```diff
@@ -9,12 +12,110 @@
...
+          A collaboration between Mameshiba and Kinako, a 2010 Japanese film based on a true story of a dog trainer,
+          meant
+          to
+          promote the latter's release the following month.
+        </p>
+      </div>
+    </div>
+  </li>
+
+ -->
```
Comment (codingcodymiller) [inline index.html]: Please remove commented code prior to submitting for review

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+$xhrRandom.open('GET', 'https://api.jikan.moe/v4/anime?sfw=true&page=' + randomPage);
+$xhrRandom.responseType = 'json';
+$xhrRandom.addEventListener('load', loading);
+
+function loading(event) {
+  random.push($xhrRandom.response);
+  var randomArray = [];
+  for (var a = 0; a < random[0].data.length; a++) {
+    randomArray.push(random[0].data[a]);
+  }
```
Comment (codingcodymiller) [inline js/main.js]: I don't understand the purpose of `random` and `randomArray`. You should be able to just choose three objects at random from `$xhrRandom.response.data` and use those objects to create your DOM trees.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by YoonHCho** [comment]
removed comments and randomArray
AJAX Feature 1

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**